client/src/main/java/org/apache/celeborn/client/ShuffleClient.java

```diff
@@ -200,7 +247,16 @@ public abstract PartitionLocation registerMapPartitionTask(
       int shuffleId, int numMappers, int mapId, int attemptId, int partitionId) throws IOException;
 
   public abstract ConcurrentHashMap<Integer, PartitionLocation> getPartitionLocation(
-      int shuffleId, int numMappers, int numPartitions);
+      int shuffleId, int appShuffleId, int numMappers, int numPartitions);
```

Comment:
Better to unify the variable names, for example `sparkShuffleId`/`appShuffleId` -> `appShuffleId`, `uniqueShuffleId`/`celebornShuffleId` -> `celebornShuffleId`

Reply:
I suggest to follow below rules for variable names:
1. in spark package, sparkShuffleId/shuffleId
2. In celeborn package, appShuffleId/shuffleId

It doesn't make sense for celebornShuffleId in celeborn package, for instance, lots of shuffleId variable name is used in LifecycleManager
but it is ok for me to combine both sparkShuffleId/appShuffleId to appShuffleId in spark/celeborn packages if you prefer

Reply:
I do prefer unify sparkShuffleId/appShuffleId to appShuffleId is better